src/App.tsx

```diff
@@ -16,80 +17,127 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPABET,
-  LENGTH,
+  NONE = 'NONE',
+  ALPABET = 'ALPABET',
+  LENGTH = 'LENGTH',
 }
 
 type ReorderOptions = {
   sortType: SortType,
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
-export function getReorderedGoods(
+export function GetReorderedGoods(
   goods: string[],
```

Comment:
it`s not ok to return jsx like this in separate function, use this function to reorder items and return reordered list, then map it insine of App component